server/src/main/java/com/google/musicanalysis/types/AnalysisGroup.java

```diff
@@ -2,8 +2,10 @@
 
 import java.util.ArrayList;
 import java.util.HashMap;
+import java.io.Serializable;
 
-public class AnalysisGroup {
+// Groups together data for easy sending/reading from front to back
```

Comment:
This should be a properly formatted javadoc comment (https://google.github.io/styleguide/javaguide.html#s7-javadoc)